docs/nodes/mamaki-testnet.md

```diff
@@ -38,6 +38,7 @@ There is a list of RPC endpoints you can use to connect to Mamaki Testnet:
 * [https://rpc-mamaki.pops.one](https://rpc-mamaki.pops.one)
 * [https://rpc-1.celestia.nodes.guru](https://rpc-1.celestia.nodes.guru)
 * [https://celestia-testnet-rpc.polkachu.com/](https://celestia-testnet-rpc.polkachu.com/)
+* [https://rpc.celestia.testnet.run](https://rpc.celestia.testnet.run/)
```

Comment:
Tried connecting on port 9090 for DA API stuff, your port is closed.

Reply:
> Tried connecting on port 9090 for DA API stuff, your port is closed.

weird, lemme check.

Reply:
> Tried connecting on port 9090 for DA API stuff, your port is closed.

hmm, can try again? -8090-

Reply:
When I try doing the following connected to your RPC:
```
root@yaz-light-node:~/celestia-node# celestia light start --core.grpc rpc.celestia.testnet.run
:8090 --keyring.accname yaz-light
```

And try to curl:
```
root@yaz-light-node:~# curl -X GET http://127.0.0.1:26658/balance/celestia18njrj7tsv3k9gnjj6
hq3nnhaggj3jam4p8a4eg
"querying client for balance: rpc error: code = Unavailable desc = connection error: desc =
\"transport: Error while dialing dial tcp 172.67.223.218:8090: i/o timeout\""root@yaz-light-
node:~# curl -X GET http://127.0.0.1:26658/balance/celestia18njrj7tsv3k9gnjj6hq3nnhaggj3jam4
p8a4eg
```

I get the same behavior with port 9090

Reply:
> 

just checked. see a problem with proxy, should  work now. with `9090`